pkg/operator/operator.go

```diff
@@ -53,24 +53,6 @@ type Operator struct {
 	namespace string
 }
 
-type newClientBuilder struct{}
```

Comment:
Ah, I figured there was a better way. Thanks for patching this!

Reply:
I think `ClientBuilder` was needed for v0.8.3, but then v0.9.0 went back to using `NewCacheFunc`.

Reply:
Right, my mistake.

Either way, glad it's simplified now!